marvel_superheroes/src/components/Hero/Hero.tsx

```diff
@@ -1,31 +1,38 @@
-import React from "react";
-import { Link } from "react-router-dom";
-import './Hero.css';
-import { IHero } from "../HomePage/HomePage";
+import React from "react"
+import { Link } from "react-router-dom"
+import "./Hero.css"
```

Comment:
better to use single quote and semicolon in the end of statements. change eslint rules